docs/source/cloud-setup/policy.rst

```diff
@@ -19,41 +19,62 @@ Example usage:
 Overview
 --------
 
-Admin policies are :ref:`implemented in Python packages <implement-admin-policy>` first, then get distributed and applied at the :ref:`server-side <server-side-admin-policy>` and/or the :ref:`client-side <client-side-admin-policy>`. If both are set, the policy at client-side will be applied first, the mutated tasks and SkyPilot config will then be processed by the policy at server-side.
+SkyPilot has a :ref:`client-server architecture <sky-api-server>`, where a centralized API server can be deployed and users can interact with the server through a client.
+
+To deploy a admin policy, here are the steps:
+
+1. :ref:`Implement the policy in a Python package <implement-admin-policy>` or :ref:`serve the policy in a RESTful server <serve-admin-policy-in-server>`.
```

Comment:
I put these two options on the same level since implementing a restful server does not necessarily require inheriting from our AdminPolicy interface. Refer to the restful server section for details.